packages/share_plus/share_plus/example/web/index.html

```diff
@@ -30,16 +30,6 @@
   <link rel="manifest" href="manifest.json">
 </head>
 <body>
-  <!-- This script installs service_worker.js to provide PWA functionality to
-       application. For more information, see:
-       https://developers.google.com/web/fundamentals/primers/service-workers -->
-  <script>
-    if ('serviceWorker' in navigator) {
-      window.addEventListener('flutter-first-frame', function () {
-        navigator.serviceWorker.register('flutter_service_worker.js');
-      });
-    }
-  </script>
-  <script src="main.dart.js" type="application/javascript"></script>
```

Comment:
Removed to not see the following message in logs:
```
Warning: In index.html:39: Manual service worker registration deprecated. Use flutter.js service worker bootstrapping instead.
```

The change is the same as in https://github.com/flutter/packages/pull/7114